cord19kg/examples/notebooks/colab/Glucose is a risk facor for COVID-19 (3000 papers).ipynb

```diff
@@ -0,0 +1,799 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "kernelspec": {
+      "display_name": "bluegraph",
+      "language": "python",
+      "name": "bluegraph"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.6.12"
+    },
+    "colab": {
+      "name": "Glucose is a risk facor for COVID-19 (3000 papers).ipynb",
+      "provenance": [],
+      "history_visible": true
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4lakIXNEwMK4"
+      },
+      "source": [
+        "To install BlueGraph and the necessary dependecies from the notebook, run the following cell."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "UbEdKOdlt925"
+      },
+      "source": [
+        "# Install bluegraph\n",
+        "! git clone https://github.com/BlueBrain/BlueGraph\n",
+        "! cd BlueGraph\n",
+        "! pip install bluegraph[cord19kg]\n",
```

Comment:
This pip install is not working for me when in colab

Reply:
I’m getting:
ERROR: Could not find a version that satisfies the requirement bluegraph[cord19kg] (from versions: none)
ERROR: No matching distribution found for bluegraph[cord19kg]

Reply:
Thanks for spotting, last commit should fix it. Note also that there is a bug at the end of the notebook (before running the vis app), which is fixed in another PR (#45 ), so If you don't mind, I would merge that one first, then sync this branch, so that you can test colab notebook properly

Reply:
Works for me